AsyncDisplayKitTests/ASStackLayoutSpecSnapshotTests.mm

```diff
@@ -21,6 +21,15 @@ @interface ASStackLayoutSpecSnapshotTests : ASLayoutSpecSnapshotTestCase
 
 @implementation ASStackLayoutSpecSnapshotTests
 
+#pragma mark - XCTestCase
+
+- (void)setUp
+{
+  [super setUp];
+  
+  self.recordMode = NO;
```

Comment:
Yes we finally should add this. By default `recordMode` is `NO` as we don't want to create any snapshot images. Setting it to `YES` will trigger to create the snapshot images.